lighthouse-core/report/html/renderer/util.js

```diff
@@ -94,6 +94,22 @@ class Util {
     }
   }
 
+  /**
+   * Poor man's implementation of i18n._formatIcuMessage()
+   * @param {{message: string, values: Object|undefined}} audit
+   * @return {string}
+   */
+  static basicMsgFormat({message, values}) {
+    let resolvedMsg = message;
+    for (const [valueName, value] of Object.entries(values)) {
+      // Look for `{valueName}` or `{valueName, number}`
+      // eslint-disable-next-line no-useless-escape
+      const re = new RegExp('\{\\s*?' + valueName + '\\b.*?\}');
```

Comment:
should this be a little narrower given we don't support everything? i.e. maybe `(,\s*number)?}`

Reply:
wfm